src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -59,7 +59,7 @@ public function getConfigTreeBuilder()
                 ->arrayNode('collection')
                     ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('order')->defaultNull()->info('The default order of results.')->end()
+                        ->scalarNode('order')->defaultValue('ASC')->info('The default order of results.')->end()
```

Comment:
Can you just add a comment to explain why we need to do this (this MySQL / Postgres "bug" is not obvious at all).